superset/models/core.py

```diff
@@ -748,6 +748,9 @@ class Database(Model, AuditMixinNullable, ImportMixin):
     """
         ),
     )
+    encrypted_extra = Column(
+        EncryptedType(Text, config.get("SECRET_KEY")), nullable=True
+    )
```

Comment:
Nit: as per #8409, config variables are assumed to always be set. So this should be changed to ...`config["SECRET_KEY"]`